File: PMUG02_16_2021_Project/Start_Java/app/src/main/java/com/neudesic/pmugfebappstart/model/Person.java
package com.neudesic.pmugfebappstart.model;

import java.time.LocalDate;

public class Person {
    private int age;
    private String name;
    private LocalDate birthday;

    public Person(int age, String name, LocalDate birthday) {
        this.age = age;
        this.name = name;
        this.birthday = birthday;
    }


    public void setAge(int age) {
        this.age = age;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setBirthday(LocalDate birthday) {
        this.birthday = birthday;
    }

    public int getAge() {
        return age;
    }

    public String getName() {
        return name;
    }

    public LocalDate getBirthday() {
        return birthday;
    }
}

File: PMUG02_16_2021_Project/Start_Kotlin/app/src/main/java/com/neudesic/pmug02_16_2021_kotlin/model/Person.kt
package com.neudesic.pmug02_16_2021_kotlin.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import androidx.room.TypeConverters
import com.neudesic.pmug02_16_2021_kotlin.database.Converter
import org.joda.time.LocalDate


@Entity(tableName = "person_table")
class Person(
        val age: Int,
        val name: String,
        @TypeConverters(Converter::class)
        val birthday: LocalDate
) {
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
}